templates/sample/src/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Blockly Demo: Generating Javascript</title>
+    <script src="../js/blockly_compressed.js"></script>
```

Comment:
see comment below - if you move the script to `static/js/` you should be able to use the blockly npm package and avoid having to do this.

Reply:
I moved this library js file to static/js/blockly folder (Last time I failed to bundle it with npm package). Also I will try to test on blockly npm package again and let you know.